src/src/components/Molecules/Portal.tsx

```diff
@@ -18,7 +18,11 @@ export function Portal({
 }: {
   readonly children: JSX.Element;
 }): JSX.Element {
-  const element = React.useMemo(() => document.createElement('div'), []);
+  const element = React.useMemo(() => {
+    let element = document.createElement('div');
+    element.className = "h-full"
```

Comment:
[non-blocking]

Could you enable the `prettier` plugin in your IDE to keep the code formatting consistent?